rclcpp_components/src/component_manager.cpp

```diff
@@ -196,6 +196,11 @@ ComponentManager::OnLoadNode(
 
       try {
         node_wrappers_[node_id] = factory->create_node_instance(options);
+      } catch (const std::exception &ex) {
```

Comment:
http://build.ros2.org/job/Rpr__rclcpp__ubuntu_focal_amd64/31/testReport/junit/rclcpp_components/uncrustify/src_component_manager_cpp/

```suggestion
      } catch (const std::exception & ex) {
```

Reply:
Fixed, thanks for checking.